_upload-artifacts.yml

```diff
@@ -6,12 +6,7 @@ steps:
 # This comes after uploading azure artifacts so we can debug the azure artifact
 # if this step fails.
 - bash: |
-    set -e
-    find "${ARTIFACT_STAGING_DIR}" \
-      -type f \
-      -name "*.tar.xz" \
-      -print0 \
-      | xargs -0 -n1 ./check-tarball.sh
+    for f in ${ARTIFACT_STAGING_DIR}/*.tar.xz; do ./check-tarball.sh "$f"; done
```

Comment:
The new for loop would have problems if `ARTIFACT_STAGING_DIR` had whitespace in it. Although that directory name is in our control, it would probably be better if it was `for f in "${ARTIFACT_STAGING_DIR}/*.tar.xz";` just to avoid readers having to thing about whether the lack of quoting may be a problem.

Unless I'm missing something, the check-tarball.sh return code is no longer checked is it? (at least, not other than the last invocation of check-tarball.sh)

Reply:
I couldn't find a good reason for the `find` (and it clearly works without it, as CI shows). Let me do a follow-up to add the quotes (I thought I had that and then saw globbing broken, but that doesn't make sense, so let's try again.)